pymc3/tests/test_distributions.py

```diff
@@ -944,17 +944,39 @@ def test_lkj(self, x, eta, n, lp):
 
     @pytest.mark.parametrize('n', [2, 3])
     def test_dirichlet(self, n):
-        self.pymc3_matches_scipy(Dirichlet, Simplex(
-            n), {'a': Vector(Rplus, n)}, dirichlet_logpdf)
+        self.pymc3_matches_scipy(
+            Dirichlet,
+            Simplex(n),
+            {'a': Vector(Rplus, n)},
+            dirichlet_logpdf
+        )
+
+    @pytest.mark.parametrize('n', [3, 4])
+    def test_dirichlet_init_fail(self, n):
+        with Model():
+            with pytest.raises(ValueError) as err:
+                _ = Dirichlet('x', a=np.zeros(n), shape=n)
+            err.match("All concentration parameters (a) must be > 0.")
```

Comment:
```suggestion
            err.match(r"All concentration parameters \(a\) must be > 0.")
```

Parenthesis are a special character in regular expressions so you have to escape them with a backlash in the `match`